src/Serilog.Sinks.Console/Serilog.Sinks.Console.csproj

```diff
@@ -4,7 +4,7 @@
     <Description>A Serilog sink that writes log events to the console/terminal.</Description>
     <VersionPrefix>4.0.0</VersionPrefix>
     <Authors>Serilog Contributors</Authors>
-    <TargetFrameworks>net45;netstandard1.3;netstandard2.0;netcoreapp1.1;netcoreapp2.0</TargetFrameworks>
+    <TargetFrameworks>net45;netstandard1.3;netstandard2.0</TargetFrameworks>
     <AssemblyName>Serilog.Sinks.Console</AssemblyName>
```

Comment:
I probably miss some context, but it seems `netcoreapp` targets are unnecessary

Reply:
:+1: - I think in a few cases these were the product of the frameworks providing a shallower dependency graph; not sure where this all stands now, good to pare it back.